Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -269,13 +269,87 @@ public final class RewardAddOnSelectionViewModel: RewardAddOnSelectionViewModelT
         )
       }
 
-    Signal.zip(project, baseReward, refTag)
+    let calculatedShippingTotal = Signal.combineLatest(
```

Comment:
I noticed this exact block of code you've added exists in the `PledgeViewModel`. You should re-factor so that both of these view models are not doing the same thing.

Reply:
They aren't entirely the same, one combines 3 signal and the other combines 2.

Reply:
I understand, it still looks like it can be cleaned up. I think you can do the work in this view model and pass the shared data to the PledgeViewModel.

Reply:
Yeah, I actually thought about that